Makefile.am

```diff
@@ -5518,6 +5518,10 @@ if BUILD_PYTHON3_BINDINGS
 		mv -f _py3hbac.so pyhbac.so ; \
 		mv -f _py3sss_murmur.so pysss_murmur.so ; \
 		mv -f _py3sss_nss_idmap.so pysss_nss_idmap.so
+endif
+if BUILD_SSH_KNOWN_HOSTS_CMD
+	cd $(DESTDIR)$(bindir) && \
+		$(LN_S) sss_ssh_authorizedkeys sss_ssh_knownhosts
```

Comment:
This command fails if the symlink already exists (make install && make install)

Reply:
This remark doesn't apply any more as the application became a separate one instead of a symbolic link.